src/status_im/data_store/messages.cljs

```diff
@@ -111,10 +112,10 @@
                            :timestamp (timestamp)})]
       (data-store/save message'))))
 
-(defn update
+(defn update 
   [{:keys [message-id] :as message}]
   (when (data-store/exists? message-id)
-    (let [message (clojure.core/update message :user-statuses vals)]
+    (let [message (update-if-present message :user-statuses vals)]
```

Comment:
The problem is, that without `update-if-present`, the message update function always assoc-ed `:user-statuses` into message record, so when your update-message didn't include `:user-statuses`, they were effectively deleted in realm-db.
It was not problematic before, because the message update function was called only for update of the user-statuses, now when we also update message preview, it would be problematic if not for this change.